### ISSUE TITLE
Model 3

### ISSUE THREAD

**Issue by jacaseyclyde** [sjsu-high-z-classify/GalNet#35]
**Model architecture.**
This is very similar to Model 2 (#32), but instead we add a Dense layer of 2048 neurons as an intermediate data reduction before the 64 neurons.

Layer | Type | Size | Features |  Kernel | Stride | Padding | Activation | Notes
--------|--------|-------|-------------|-----------|--------|------------|-------------|---------
In | Input | 69x69 | 3 | | | | | Input Image
1 | Conv2D | 69x69 | 32 | 6x6 | 1 | | Same | ReLU |
2 | Conv2D | 69x69 | 64 | 5x5 | 1 | Same | | | ReLU |
3 | MaxPool2D | 34x34 | 64 | 2x2 | 2 | Valid | | |
4 | Conv2D | 34x34 | 128 | 2x2 | 1 | Same | | ReLU |
5 | MaxPool2D | 17x17 | 64 | 2x2 | 2 | Valid | | |
6 | Conv2D | 17x17 | 128 | 3x3 | 1 | Same | | ReLU |
7 | Dense | 36,992 | | | | | | Flattened layer
8 | Dense | 2048 | | | | | ReLU |
9 | Dense | 64 | | | | | ReLU |
Out | Dense | 3 | | | | | Softmax | Output prediction

**Hyperparameters**
Initial learning rate: 0.0001
Optimizer: Nadam
Loss: crossentropy
Loss Weights: None
Metrics: accuracy

**Additional context**
Use dropout=.5 on all hidden Dense layers.

**References**
<to be updated>
